site/docs/expressions/extended_expression.md

```diff
@@ -1,19 +1,25 @@
 # Extended expression
 
-Extended expression is provided for expression level protocol instead of plan rels. It mainly targets for expression only evaluation, such as those computed in Filter/Project/Aggregation rels. Different from original expression defined in substrait protocol, it requires more information to completely describe the computation context, including input data schema, referred function signatures and output schema. 
+Extended expressions are a top-level message, like a plan, but for expressions. They can be used to pass around expressions for use in filter, projection, and aggregation operations that aren't part of a full plan. Because they are a top-level message, they include more context than expressions that exist _within_ a plan. Some of this is additional computational context, such as the input data schema and the output schema. And some are message metadata, such as the Substrait version and the URIs of referenced extensions.
```

Comment:
@wjones127 just a nit

```suggestion
Extended expressions are a top-level message, like a plan, but for expressions. They can pass around expressions to be used in filter, projection, and aggregation operations that aren't part of a full plan. Because they are a top-level message, they include more context than expressions that exist _within_ a plan. Some of this is additional computational context, such as the input data schema and the output schema. And some are message metadata, such as the Substrait version and the URIs of referenced extensions.
```